pyvelox/test/test_signatures.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import pyvelox.pyvelox as pv
+
+
+class TestFunctionSignatures(unittest.TestCase):
+    def test_get_signatures(self):
```

Comment:
nit: could move the first few lines to a different `test_clear_signatures()` test case, so that if it fails it gives you a more descriptive message.